libnl-route-3.sym

```diff
@@ -1111,6 +1111,7 @@ global:
 	rtnl_neigh_get_master;
 	rtnl_neigh_set_master;
 	rtnl_netem_set_delay_distribution_data;
+	rtnl_qdisc_get_by_kind;
```

Comment:
we must never modify existing section in the `.sym` file, after they are released. That would completely mess up symbol versioning.

This needs to go to a new section `libnl_3_6`.

Reply:
Moved to new _libnl_3_6_ section.